packages/salesforcedx-vscode-soql/src/queryDataView/queryDataViewService.ts

```diff
@@ -50,11 +53,18 @@ export class QueryDataViewService {
   }
 
   private updateWebviewWith(queryData: QueryResult<JsonMap>) {
-    this.currentPanel?.webview.postMessage({
-      type: 'update',
-      data: queryData,
-      documentName: getDocumentName(this.document)
-    });
+    this.currentPanel?.webview
+      .postMessage({
+        type: 'update',
+        data: queryData,
+        documentName: getDocumentName(this.document)
+      })
+      .then(undefined, async (err: string) => {
+        const errorType = 'data_view_post_message';
```

Comment:
Again, i've never been able to trigger this theoretical exception with communicating across webview/iframe.